recorder_test.go

```diff
@@ -2,28 +2,16 @@ package zipkintracer
 
 import (
 	"sync/atomic"
-	"testing"
-	"time"
 
-	"github.com/stretchr/testify/assert"
+	"github.com/openzipkin/zipkin-go/model"
 )
 
-func TestInMemoryRecorderSpans(t *testing.T) {
-	recorder := NewInMemoryRecorder()
-	var apiRecorder SpanRecorder = recorder
-	span := RawSpan{
-		Context:   SpanContext{},
-		Operation: "test-span",
-		Start:     time.Now(),
-		Duration:  -1,
-	}
-	apiRecorder.RecordSpan(span)
-	assert.Equal(t, []RawSpan{span}, recorder.GetSpans())
-	assert.Equal(t, []RawSpan{}, recorder.GetSampledSpans())
-}
-
 type CountingRecorder int32
 
-func (c *CountingRecorder) RecordSpan(r RawSpan) {
+func (c *CountingRecorder) Send(span model.SpanModel) {
```

Comment:
previous name is less distracting unless somehow the word send is interesting here. the type is recorder right?

Reply:
yes and we changed the name to fulfill the interface.